### ISSUE TITLE
Fedora Silverblue (propably also Kinoite) are not recognized and are without an icon

### ISSUE THREAD

**Issue by inffy** [vinceliuice/grub2-themes#186]
Fedora Silverblue is not recognized by the themes and so are shown without on icon.

I think we could just use the Fedora logo on these too, or specific logo for Silverblue?

**Comment by vinceliuice** [comment]
Could you upload your `/boot/grub2/grub.cfg` to me? then I can know the  Silverblue icon ID

**Comment by inffy** [comment]
[grub.txt](https://github.com/vinceliuice/grub2-themes/files/10047392/grub.txt)

here

**Comment by vinceliuice** [comment]
Oh no, fedora seems not put the boot option in `/boot/grub2/grub.cfg` , it's in here:  `/boot/loader/entries`
I need the ID name after `grub_class`

**Comment by inffy** [comment]
here is /boot/loader/entries/ostree-1-fedora.conf
[file1.txt](https://github.com/vinceliuice/grub2-themes/files/10047530/file1.txt)

/boot/loader/grub.cfg
[grub2.txt](https://github.com/vinceliuice/grub2-themes/files/10047532/grub2.txt)

Seems these might not give any help though :/

**Comment by vinceliuice** [comment]
I did not see any grub_class in ostree-1-fedora.conf, maybe that's the reason grub2 not show the system logo

**Comment by vinceliuice** [comment]
This is my Fedora 37 boot entry:
```
title Fedora Linux (5.19.15-301.fc37.x86_64) 37 (Workstation Edition)
version 5.19.15-301.fc37.x86_64
linux /boot/vmlinuz-5.19.15-301.fc37.x86_64
initrd /boot/initramfs-5.19.15-301.fc37.x86_64.img
options root=UUID=f0d79a71-b96b-4093-8b21-739f6644dd5f ro rd.driver.blacklist=nouveau modprobe.blacklist=nouveau nvidia-drm.modeset=1 initcall_blacklist=simpledrm_platform_driver_init rhgb quiet rd.driver.blacklist=nouveau modprobe.blacklist=nouveau nvidia-drm.modeset=1 initcall_blacklist=simpledrm_platform_driver_init 
grub_users $grub_users
grub_arg --unrestricted
grub_class fedora
```

**Comment by vinceliuice** [comment]
There is `grub_class fedora` that's the id theme use for icon name

**Comment by inffy** [comment]
oh well, no idea where it might store it, if it even stores it somewhere.

**Comment by trytomakeyouprivate** [comment]
Yes Kinoite has no icon too. Just setting it as the Fedora Icon should be a perfectly fine workaround.
